languages/common-lisp/exercises/concept/sameness/sameness-test.lisp

```diff
@@ -0,0 +1,149 @@
+;; Ensures that sameness.lisp and the testing library are always loaded
+(eval-when (:compile-toplevel :load-toplevel :execute)
+  ;; (load "sameness")
+  (load "./.meta/example.lisp")
+  (ql:quickload :fiveam))
+
+;; Defines the testing package with symbols from sameness and FiveAM in scope
+;; The `run-tests` function is exported for use by both the user and test-runner
+(defpackage sameness-test
+  (:use :cl :fiveam :sameness)
+  (:export :run-tests))
+
+;; Enter the testing package
+(in-package :sameness-test)
+
+;; Define and enter a new FiveAM test-suite
+(def-suite sameness-suite)
+(in-suite sameness-suite)
+
+(defparameter +an-array+ #(1 2 3))
```

Comment:
I chose not to use `defconstant` since reloading the file will drop you into the debugger - so for myself while developing it and for the student while doing it (not on the web i guess) I chose `defparameter`.

Reply:
Okay, I think that's reasonable! I hadn't run into that, but I was just copying it into the REPL. Playing with it now, I see that it tries to optimise it away and complains about things not being fully defined.